src/libsync/owncloudpropagator.cpp

```diff
@@ -115,6 +124,61 @@ static bool blacklistCheck(SyncJournalDb* journal, const SyncFileItem& item)
     return newEntry.isValid();
 }
 
+void PropagateItemJob::itemDone(SyncFileItemPtr item, SyncFileItem::Status status, const QString &errorString)
```

Comment:
No, in this case we cannot do that, otherwise you will need to overwrite "_item" each time you do "done()". This need to be like that. Please also mind that _item in this case is the whole Bundle.